src/logic/quickstart.utils.ts

```diff
@@ -1,9 +1,60 @@
+import { ProfileDocumentNode } from '@superfaceai/ast';
 import { SuperJson } from '@superfaceai/one-sdk';
 import { join as joinPath } from 'path';
 
-import { EXTENSIONS } from '..';
+import { EXTENSIONS, getProfileDocument } from '..';
 import { exists, readdir } from '../common/io';
 
+export async function loadProfileAst(
+  superJson: SuperJson,
+  profile: { profile: string; scope: string; version?: string }
```

Comment:
What if: no scope (360)?

Reply:
You mean what will happen if there is a profile without scope? It will probably explode 🤯

Reply:
that's not gud